prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0014.fhir

```diff
@@ -1,8 +1,8 @@
 {
   "resourceType" : "Bundle",
-  "id" : "1666796912633094300.43c7bfca-43db-4519-8ad8-4ecb454ef043",
+  "id" : "1671808447500138700.d15e4594-2c9c-445a-a5bb-2e80a96a1ade",
   "meta" : {
-    "lastUpdated" : "2022-10-26T11:08:32.720-04:00"
+    "lastUpdated" : "2022-12-23T10:14:07.559-05:00"
   },
   "identifier" : {
     "value" : "MT_COCZS_ORU_ZSPHELR.1.6384446"
```

Comment:
It's actually amazing how much code reduction the changes bring into these files!